frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1124,22 +1520,9 @@ function ReaderFooter:addToMainMenu(menu_items)
             },
             getMinibarOption("all_at_once", self.updateFooterTextGenerator),
             {
-                text = _("Hide empty items"),
-                help_text = _([[This will hide values like 0 or off.]]),
-                enabled_func = function()
-                    return self.settings.all_at_once == true
-                end,
-                checked_func = function()
-                    return self.settings.hide_empty_generators == true
-                end,
-                callback = function()
-                    self.settings.hide_empty_generators = not self.settings.hide_empty_generators
-                    self:refreshFooter(true, true)
-                end,
-            },
-            getMinibarOption("reclaim_height"),
-            {
-                text = _("Auto refresh"),
+                text = _("Auto refresh items"),
+                help_text = _("This option allows certain items to update without needing a full-page update. For example, the time"..
+                " item will update every minute regardless of user input."),
```

Comment:
I'm afraid that's bad for translation.
```suggestion
                help_text = _("This option allows certain items to update without needing a full-page update. For example, the time item will update every minute regardless of user input."),
```